localstack/utils/cloudformation/template_deployer.py

```diff
@@ -516,6 +519,52 @@ def get_apigw_resource_params(params, **kwargs):
                 'InstanceIds': lambda params, **kw: [kw['resources'][kw['resource_id']]['PhysicalResourceId']]
             }
         }
+    },
+    'EC2::VPC': {
```

Comment:
We already have the deploy template for creation in `EC2VPC.get_deploy_templates(...)`. Let's merge the code here (specifically the `delete` part) into the `EC2VPC.get_deploy_templates` function.